src/lib/CollectStore.js

```diff
@@ -55,7 +55,7 @@ export default class CollectStore {
       this.fetchAllAccounts(),
       this.fetchInstalledKonnectors(),
       this.fetchKonnectorResults(),
-      this.fetchKonnectorRunningJobs(domain)
+      this.fetchKonnectorUnfinishedJobs(domain)
```

Comment:
Oops, I forgot to replace here, that could be an issue before I think, nice shot 👍